README.md

```diff
@@ -45,7 +53,13 @@ const schema = {
 
 fastify
   .get('/', schema, function (req, reply) {
-    reply.send({ hello: 'world' })
+    let response = 'world'
+
+    if (req.query.name) response = req.query.name
+
+    while(req.query.excitement--) response += '!'
+
+    reply.send({ hello: response })
```

Comment:
This should not be in the example, we need to document it in https://github.com/fastify/fastify#route.